src/components/uploader/css/Uploader.css

```diff
@@ -241,7 +241,7 @@ tr.fl-uploader-file-state-uploaded button:before {
     display: block;
     overflow: hidden;
     height: 2em;
-    width: 7em;
+    width: 9em;
```

Comment:
if width is 9em then output is [https://www.dropbox.com/s/vmkl45ftt36inf4/Screenshot%20from%202016-03-17%2023%3A44%3A40.png?dl=0]
if width is 8em then output is [https://www.dropbox.com/s/s91r5engrcdxsh7/Screenshot%20from%202016-03-17%2023%3A44%3A21.png?dl=0]